packages/web/src/components/utils/CustomToast/defaultSuccessToastProps.ts

```diff
@@ -0,0 +1,8 @@
+import { UseToastOptions } from '@chakra-ui/react';
+
+export const defaultSuccessToastProps: UseToastOptions = {
+  status: 'success',
+  position: 'top',
+  duration: 30 * 1000,
```

Comment:
5 seconds for a success message is sufficient
```suggestion
  duration: 5 * 1000,
```